<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>80</width>
    <height>25</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>kakoune-qt</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources.qrc">
    <normaloff>:/images/resources/images/kakoune_logo_32.png</normaloff>:/images/resources/images/kakoune_logo_32.png</iconset>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item row="1" column="0">
     <widget class="QTextEdit" name="textEdit">
      <property name="frameShape">
       <enum>QFrame::WinPanel</enum>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAsNeeded</enum>
      </property>
      <property name="tabChangesFocus">
       <bool>false</bool>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="acceptRichText">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QTextEdit" name="lineEdit">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>18</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="lineWrapMode">
       <enum>QTextEdit::NoWrap</enum>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="acceptRichText">
       <bool>false</bool>
      </property>
      <property name="textInteractionFlags">
       <set>Qt::NoTextInteraction</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_as"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
    <widget class="QMenu" name="menuUI_options">
     <property name="title">
      <string>UI options</string>
     </property>
     <addaction name="actionStatus_bar_placement"/>
     <addaction name="actionAssistant"/>
    </widget>
    <addaction name="menuUI_options"/>
   </widget>
   <widget class="QMenu" name="menuBuffer">
    <property name="title">
     <string>Buffer</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <widget class="QMenu" name="menuDocumentation">
     <property name="title">
      <string>Documentation</string>
     </property>
     <addaction name="actionCommands"/>
     <addaction name="actionExec_eval"/>
     <addaction name="actionExpansions"/>
     <addaction name="actionFaces"/>
     <addaction name="actionHighlighters"/>
     <addaction name="actionHooks"/>
     <addaction name="actionOptions"/>
     <addaction name="actionRegisters"/>
     <addaction name="actionShortcuts"/>
    </widget>
    <addaction name="menuDocumentation"/>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="separator"/>
    <addaction name="actionCut"/>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
    <addaction name="separator"/>
    <addaction name="actionSearch"/>
    <addaction name="actionNext_match"/>
    <addaction name="actionPrevious_match"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuOptions"/>
   <addaction name="menuBuffer"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <property name="floatable">
    <bool>true</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
   <addaction name="actionSave_as"/>
   <addaction name="separator"/>
   <addaction name="actionUndo"/>
   <addaction name="actionRedo"/>
   <addaction name="separator"/>
   <addaction name="actionCut"/>
   <addaction name="actionCopy"/>
   <addaction name="actionPaste"/>
   <addaction name="separator"/>
   <addaction name="actionSearch"/>
   <addaction name="actionNext_match"/>
   <addaction name="actionPrevious_match"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionQuit">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/tango_icons/tango-icon-theme-0.8.90/32x32/actions/system-shutdown.png</normaloff>:/tango_icons/tango-icon-theme-0.8.90/32x32/actions/system-shutdown.png</iconset>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/tango_icons/tango-icon-theme-0.8.90/32x32/actions/document-open.png</normaloff>:/tango_icons/tango-icon-theme-0.8.90/32x32/actions/document-open.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionStatus_bar_placement">
   <property name="text">
    <string>Status bar placement</string>
   </property>
  </action>
  <action name="actionAssistant">
   <property name="text">
    <string>Assistant</string>
   </property>
  </action>
  <action name="actionCommands">
   <property name="text">
    <string>Commands</string>
   </property>
  </action>
  <action name="actionExec_eval">
   <property name="text">
    <string>Exec / eval</string>
   </property>
  </action>
  <action name="actionExpansions">
   <property name="text">
    <string>Expansions</string>
   </property>
  </action>
  <action name="actionFaces">
   <property name="text">
    <string>Faces</string>
   </property>
  </action>
  <action name="actionHighlighters">
   <property name="text">
    <string>Highlighters</string>
   </property>
  </action>
  <action name="actionHooks">
   <property name="text">
    <string>Hooks</string>
   </property>
  </action>
  <action name="actionOptions">
   <property name="text">
    <string>Options</string>
   </property>
  </action>
  <action name="actionRegisters">
   <property name="text">
    <string>Registers</string>
   </property>
  </action>
  <action name="actionShortcuts">
   <property name="text">
    <string>Shortcuts</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/tango_icons/tango-icon-theme-0.8.90/32x32/actions/document-save.png</normaloff>:/tango_icons/tango-icon-theme-0.8.90/32x32/actions/document-save.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/tango_icons/tango-icon-theme-0.8.90/32x32/actions/document-save-as.png</normaloff>:/tango_icons/tango-icon-theme-0.8.90/32x32/actions/document-save-as.png</iconset>
   </property>
   <property name="text">
    <string>Save as</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/tango_icons/tango-icon-theme-0.8.90/32x32/actions/edit-undo.png</normaloff>:/tango_icons/tango-icon-theme-0.8.90/32x32/actions/edit-undo.png</iconset>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/tango_icons/tango-icon-theme-0.8.90/32x32/actions/edit-redo.png</normaloff>:/tango_icons/tango-icon-theme-0.8.90/32x32/actions/edit-redo.png</iconset>
   </property>
   <property name="text">
    <string>Redo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/tango_icons/tango-icon-theme-0.8.90/32x32/actions/edit-cut.png</normaloff>:/tango_icons/tango-icon-theme-0.8.90/32x32/actions/edit-cut.png</iconset>
   </property>
   <property name="text">
    <string>Cut</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/tango_icons/tango-icon-theme-0.8.90/32x32/actions/edit-copy.png</normaloff>:/tango_icons/tango-icon-theme-0.8.90/32x32/actions/edit-copy.png</iconset>
   </property>
   <property name="text">
    <string>Copy</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/tango_icons/tango-icon-theme-0.8.90/32x32/actions/edit-paste.png</normaloff>:/tango_icons/tango-icon-theme-0.8.90/32x32/actions/edit-paste.png</iconset>
   </property>
   <property name="text">
    <string>Paste</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionSearch">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/tango_icons/tango-icon-theme-0.8.90/32x32/actions/system-search.png</normaloff>:/tango_icons/tango-icon-theme-0.8.90/32x32/actions/system-search.png</iconset>
   </property>
   <property name="text">
    <string>Search</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionNext_match">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/tango_icons/tango-icon-theme-0.8.90/32x32/actions/go-next.png</normaloff>:/tango_icons/tango-icon-theme-0.8.90/32x32/actions/go-next.png</iconset>
   </property>
   <property name="text">
    <string>Next match</string>
   </property>
  </action>
  <action name="actionPrevious_match">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/tango_icons/tango-icon-theme-0.8.90/32x32/actions/go-previous.png</normaloff>:/tango_icons/tango-icon-theme-0.8.90/32x32/actions/go-previous.png</iconset>
   </property>
   <property name="text">
    <string>Previous match</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
